Src/Fido2/AttestationFormat/FidoU2f.cs

```diff
@@ -36,15 +36,9 @@ public override (AttestationType, X509Certificate2[]) Verify(VerifyAttestationRe
         var pubKey = attCert.GetECDsaPublicKey()!;
         var keyParams = pubKey.ExportParameters(false);
 
-        if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
```

Comment:
Can't remember the case we used this for, but I guess we will notice if something breaks.